permissions/plugin-offlineonfailure-plugin.yml

```diff
@@ -3,4 +3,5 @@ name: "offlineonfailure-plugin"
 github: "jenkinsci/offlineonfailure-plugin"
 paths:
 - "org/jenkins-ci/plugins/offlineonfailure-plugin"
-developers: []
\ No newline at end of file
+developers:
+- "johnlengeling"
```

Comment:
Please make sure this is your login that you use for Jenkins infra (Jira, Artifactory, etc) and not your GitHub username.

```
Exception in thread "main" java.lang.IllegalStateException: User name not known to Artifactory: johnlengeling
```

Reply:
@slide,  I had previously logged into Artifactory right before opening the PR.  I re-verified that I could login to Artifactory, Jira, Confluence using the account above in the PR.